formats/cbor/commonMain/src/kotlinx/serialization/cbor/Cbor.kt

```diff
@@ -371,19 +372,18 @@ public class Cbor(
         private const val HEADER_ARRAY: Int = 0b100_00000
         private const val HEADER_MAP: Int = 0b101_00000
 
+        @Deprecated(
+            message = "Deprecated for removal in the favour of user-defined instances or Cbor companion object",
+            level = DeprecationLevel.ERROR,
+            replaceWith = ReplaceWith("Cbor")
```

Comment:
Wouldn't application of such replacement result in `Cbor.plain` -> `Cbor.Cbor`?

Reply:
It won't, I've checked it before the commit (and actually replaced it in tests)